From 02222928cb410d5455c397e5a5224916e575dc66 Mon Sep 17 00:00:00 2001
From: Shijith Thotton <sthotton@marvell.com>
Date: Fri, 13 Jan 2023 13:03:57 +0530
Subject: [PATCH 160/955] ci: refresh event dpdk event perf numbers

cn10k numbers are higher as we moved to A1 board with higher frequency.
Refreshed cn9k numbers.

Signed-off-by: Shijith Thotton <sthotton@marvell.com>
Change-Id: I3f7701c7e9f1f19006a1bdf50b54eb649c705038
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/94461
Reviewed-by: Ashwin Sekhar T K <asekhar@marvell.com>
Tested-by: Ashwin Sekhar T K <asekhar@marvell.com>
---
 .../cnxk-tests/event_perf/cnxk_event_perf.sh  |  2 +-
 .../test/cnxk-tests/event_perf/meson.build    |  2 +-
 ...1000.csv => cn106xx_rclk2500_sclk1100.csv} | 54 +++++++++----------
 .../ref_numbers/cn96xx_rclk2200_sclk1100.csv  | 28 +++++-----
 4 files changed, 43 insertions(+), 43 deletions(-)
 rename marvell-ci/test/cnxk-tests/event_perf/ref_numbers/{cn106xx_rclk2000_sclk1000.csv => cn106xx_rclk2500_sclk1100.csv} (82%)

diff --git a/marvell-ci/test/cnxk-tests/event_perf/cnxk_event_perf.sh b/marvell-ci/test/cnxk-tests/event_perf/cnxk_event_perf.sh
index 3d0ee1379e883..426b92d9cc8fc 100755
--- a/marvell-ci/test/cnxk-tests/event_perf/cnxk_event_perf.sh
+++ b/marvell-ci/test/cnxk-tests/event_perf/cnxk_event_perf.sh
@@ -20,7 +20,7 @@ TARGET_SSH_CMD="$TARGET_SSH_CMD -n"
 LD_LIBRARY_PATH=${LD_LIBRARY_PATH:-"/tmp/dpdk/deps/lib"}
 RES_SEPERATOR="------------------------------------------------------------------------"
 if [[ $PLAT == cn10k ]]; then
-	REF_FILE=${REF_FILE:-ref_numbers/cn106xx_rclk2000_sclk1000.csv}
+	REF_FILE=${REF_FILE:-ref_numbers/cn106xx_rclk2500_sclk1100.csv}
 	CPT_DEV=${CPT_DEV:-$(lspci -d :a0f3 | head -1 | awk -e '{ print $1 }')}
 else
 	REF_FILE=${REF_FILE:-ref_numbers/cn96xx_rclk2200_sclk1100.csv}
diff --git a/marvell-ci/test/cnxk-tests/event_perf/meson.build b/marvell-ci/test/cnxk-tests/event_perf/meson.build
index 9cda1fd7d29a5..dcdbb44ba2348 100644
--- a/marvell-ci/test/cnxk-tests/event_perf/meson.build
+++ b/marvell-ci/test/cnxk-tests/event_perf/meson.build
@@ -17,7 +17,7 @@ test_dir = meson.current_build_dir()
 run_command(copy_data, test_script)
 run_command(copy_data, 'cnxk_event_perf_gen.sh')
 run_command(copy_data, 'ref_numbers/cn96xx_rclk2200_sclk1100.csv')
-run_command(copy_data, 'ref_numbers/cn106xx_rclk2000_sclk1000.csv')
+run_command(copy_data, 'ref_numbers/cn106xx_rclk2500_sclk1100.csv')
 
 # Add the meson test
 test(test_name,
diff --git a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2000_sclk1000.csv b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
similarity index 82%
rename from marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2000_sclk1000.csv
rename to marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
index d667226037417..151778431672d 100644
--- a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2000_sclk1000.csv
+++ b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn106xx_rclk2500_sclk1100.csv
@@ -1,63 +1,63 @@
 L2FWD_EVENT (atomic queue) output in packets per second
 cores, RESULT
-1, 6681726
-8, 34808913
-16, 50666281
+1, 7314495
+8, 44187476
+16, 85869138
 
 ------------------------------------------------------------------------
 L3FWD_EVENT (ordered queue) output in packets per second
 cores, RESULT
-1, 7179770
-8, 35185270
-16, 50624874
+1, 7495352
+8, 44160021
+16, 85651417
 
 ------------------------------------------------------------------------
 PERF_ATQ (parallel queue) output in packets per second
 cores, RESULT
-1, 9.839
-8, 53.152
-16, 74.383
+1, 11.868
+8, 56.318
+16, 93.648
 
 ------------------------------------------------------------------------
 PIPELINE_ATQ_TX_FIRST (ordered queue) output in packets per second
 cores, RESULT
-1, 5.964
-8, 37.741
-16, 68.824
+1, 7.480
+8, 43.069
+16, 83.425
 
 ------------------------------------------------------------------------
 GW_MODE_NO_PREF (atomic queue) output in packets per second
 cores, RESULT
-1, 4.305
-8, 27.706
-16, 52.919
+1, 5.087
+8, 31.616
+16, 61.556
 
 ------------------------------------------------------------------------
 GW_MODE_PREF (atomic queue) output in packets per second
 cores, RESULT
-1, 5.973
-8, 37.734
-16, 68.772
+1, 7.777
+8, 43.035
+16, 83.120
 
 ------------------------------------------------------------------------
 GW_MODE_WFE_PREF (atomic queue) output in packets per second
 cores, RESULT
-1, 5.976
-8, 37.740
-16, 68.819
+1, 7.775
+8, 43.019
+16, 83.129
 
 ------------------------------------------------------------------------
 VECTOR_MODE (atomic queue) output in packets per second
 cores, RESULT
-1, 67.449
-8, 76.963
-16, 76.934
+1, 83.632
+8, 87.804
+16, 87.428
 
 ------------------------------------------------------------------------
 CRYPTO_ADAPTER_FWD (parallel queue) output in packets per second
 cores, RESULT
-1, 6.404
-8, 34.371
-16, 29.787
+1, 7.148
+8, 41.241
+16, 36.201
 
 ------------------------------------------------------------------------
diff --git a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn96xx_rclk2200_sclk1100.csv b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn96xx_rclk2200_sclk1100.csv
index 94f2d2e85eeb2..2643f2e874083 100644
--- a/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn96xx_rclk2200_sclk1100.csv
+++ b/marvell-ci/test/cnxk-tests/event_perf/ref_numbers/cn96xx_rclk2200_sclk1100.csv
@@ -1,35 +1,35 @@
 L2FWD_EVENT (ordered queue) output in packets per second
 cores, RESULT
-1, 9096509
-8, 65494261
-16, 89499435
+1, 9367017
+8, 68210627
+16, 88761668
 
 ------------------------------------------------------------------------
 L3FWD_EVENT (ordered queue) output in packets per second
 cores, RESULT
-1, 8649640
-8, 60791425
-16, 89527491
+1, 8389920
+8, 64268011
+16, 88734720
 
 ------------------------------------------------------------------------
 PERF_ATQ (parallel queue) output in packets per second
 cores, RESULT
-1, 8.570
-8, 58.039
-16, 86.567
+1, 8.289
+8, 57.647
+16, 87.201
 
 ------------------------------------------------------------------------
 PIPELINE_ATQ_TX_FIRST (atomic queue) output in packets per second
 cores, RESULT
-1, 9.669
-8, 70.783
+1, 9.849
+8, 73.211
 16, 110.001
 
 ------------------------------------------------------------------------
 CRYPTO_ADAPTER_FWD (parallel queue) output in packets per second
 cores, RESULT
-1, 3.651
-8, 20.302
-16, 21.744
+1, 4.207
+8, 16.903
+16, 26.950
 
 ------------------------------------------------------------------------
-- 
2.25.1

